tensorrt_llm/_torch/auto_deploy/utils/node_utils.py

```diff
@@ -264,3 +275,30 @@ def identify_regions_between_residuals(gm: GraphModule) -> List[Node]:
     boundary_nodes.append(output_node)
 
     return boundary_nodes
+
+
+def add_new_parameter_to_submodule(
+    gm: GraphModule, new_submodule_name: str, new_param_name: str, new_param: torch.Tensor
+) -> str:
```

Comment:
Let's use this function to keep it more consistent with other nn.Module functions:
```suggestion
def register_parameter(mod: nn.Module, target: str, param: Optional[nn.Parameter]) -> str:
```

Reply:
Sorry, I don't get this. Do you suggest to change `add_new_parameter_to_submodule`'s signature into `def register_parameter(mod: nn.Module, target: str, param: Optional[nn.Parameter]) -> str:`?

Or changing `submodule.register_parameter(new_param_name, new_param)` into sth like `register_parameter(submodule, new_param_name, new_param)`?